VDA-XP-Reward/test/didHelper.ts

```diff
@@ -0,0 +1,38 @@
+import { VeridaDIDRegistry } from "@verida/did-registry-contract/typechain-types"
+import EncryptionUtils from "@verida/encryption-utils"
+import { ethers } from "hardhat"
+
+export const endPoints = [
```

Comment:
What's the code in this file doing?

I'm confused about the need for `endPoints`.

Reply:
In the `XPReward` contract, it checks whether the `didAddress` is registered in the `DIDRegistry` contract.
To register a did in the `DIDRegistry` contract, we need endpoints.
These endpoints are used to register a DID in the `DIDRegistry` contract.

Reply:
I assume this will now be removed as we are removing DIDRegistry from this contract.

Reply:
Removed all unnecessary codes related to the `DIDRegistry` contract.